information-schema/information-schema-table-storage-stats.md

```diff
@@ -59,5 +59,5 @@ Fields in the `TABLE_STORAGE_STATS` table are described as follows:
 * `PEER_COUNT`: The number of replicas of the table.
 * `REGION_COUNT`: The number of Regions.
 * `EMPTY_REGION_COUNT`: The number of Regions that do not contain data in this table.
-* `TABLE_SIZE`: The total size of the table, in the unit of MiB.
+* `TABLE_SIZE`: The total size of the table, in the unit of MiB. Note that this value is a logical estimate based on table statistics and does not represent the actual compressed physical size in storage.
```

Comment:
The fact that this is based on statistics is good. However that it doesn't represent the "actual compressed physical size in storage" is not because the statistics aren't precise enough. This is because the size in storage and is measuring something different.

Reply:
So it represent the statistical size of the uncompressed data? With or without possible changed rows or tombstones?